crates/re_types/src/components/name_ext.rs

```diff
@@ -29,3 +30,9 @@ impl std::borrow::Borrow<str> for Name {
         self.as_str()
     }
 }
+
+impl Default for Name {
+    fn default() -> Self {
+        Self("<name>".into())
```

Comment:
Add a comment explaining why this is a good default, and not e.g. the empty string